src/bramble/client/ProjectStats.js

```diff
@@ -0,0 +1,263 @@
+/* eslint
```

Comment:
@humphd I see your point, that why earlier I said:

>  I added some linting rules just to make review less painless, they could be removed later if you wish

The point of this linting is not to introduce my own new rules to separate files. On the contrary, my goal is to reduce review time and effort that is needed from you. The rules in my file are based on suggestions you gave me throughout this review.